Corona-Warn-App/src/main/res/values/colors.xml

```diff
@@ -28,6 +28,7 @@
     <color name="colorCertificateSelectorButtonStroke">#B9BABA</color>
     <color name="colorCertificateCards">#FFFFFF</color>
     <color name="colorCertificateUserSwitch">#F5F5F5</color>
+    <color name="colorRampDownCard">#FFF7CB</color>
```

Comment:
There is another color for Dark mode, please add it

Reply:
Added other colors